cves/CVE-2016-6795.yml

```diff
@@ -69,9 +69,9 @@ fixes:
    - commit: c1869f4989942dd33fa4e189e0ac1f766fb5ac14
      note: Fix in trunk
 vccs:
-  - commit:
+  - commit: 8e67b9144aa643769b261e2492cb561e04d016ab
     note:
-  - commit:
+  - commit: 352fdb530080d04c6d1176c9d910120160000a6b
```

Comment:
What was the commit message / was something specifically broken by this commit, or was it just implemented wrong initially?